src/test/java/testutils/HttpClientThrowsExceptionOnNestedInstitutionQueries.java

```diff
@@ -0,0 +1,82 @@
+package testutils;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static testutils.HttpClientGetsNestedInstitutionResponse.ADMINISTRATION_ONE_LH_URI;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandler;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import no.unit.nva.institution.proxy.exception.InvalidUriException;
+import no.unit.nva.institution.proxy.utils.Language;
+import org.mockito.invocation.InvocationOnMock;
+
+public class HttpClientThrowsExceptionOnNestedInstitutionQueries {
```

Comment:
Client that simulates a failure on an exact moment. This client succeeds for all queries except for when trying to get a specific subsubunit.